test/solidity/Periphery/LiFiDEXAggregator.t.sol

```diff
@@ -0,0 +1,320 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.17;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IVelodromeV2Router } from "lifi/Interfaces/IVelodromeV2Router.sol";
+import { IVelodromeV2PoolCallee } from "lifi/Interfaces/IVelodromeV2PoolCallee.sol";
+import { LiFiDEXAggregator } from "lifi/Periphery/LiFiDEXAggregator.sol";
+import { TestBase } from "../utils/TestBase.sol";
+
+contract MockVelodromeV2FlashLoanCallbackReceiver is IVelodromeV2PoolCallee {
+    event HookCalled(
+        address sender,
+        uint256 amount0,
+        uint256 amount1,
+        bytes data
+    );
+
+    function hook(
+        address sender,
+        uint256 amount0,
+        uint256 amount1,
+        bytes calldata data
+    ) external {
+        emit HookCalled(sender, amount0, amount1, data);
+    }
+}
+
+contract LiFiDexAggregator is TestBase {
+    IVelodromeV2Router internal constant VELODROME_V2_ROUTER =
+        IVelodromeV2Router(0xa062aE8A9c5e11aaA026fc2670B0D65cCc8B2858); // optimism router
+    address internal constant VELODROME_V2_FACTORY_REGISTRY =
+        0xF1046053aa5682b4F9a81b5481394DA16BE5FF5a;
+    IERC20 internal constant USDC_TOKEN =
```

Comment:
you dont need this variable since you already defined it in TestBase as `ADDRESS_USDC_OPTIMISM`